tests/sentry/lang/javascript/test_processor.py

```diff
@@ -923,3 +923,69 @@ def test_node_modules_file_with_source_but_no_map_records_error(self, mock_disco
         # now we have an error
         assert len(processor.cache.get_errors(abs_path)) == 1
         assert processor.cache.get_errors(abs_path)[0] == {"url": map_url, "type": "js_no_source"}
+
+class HandlesFrameTest(TestCase):
+    def test_has_no_context_line(self):
+        project = self.create_project()
+        processor = JavaScriptStacktraceProcessor(data={}, stacktrace_infos=None, project=project)
+
+        frame = {
+            "abs_path": "http://example.com/foo.js",
+            "filename": "foo.js",
+            "platform": "javascript"
+        }
+        assert processor.handles_frame(frame, {}) is True
+
+    def test_has_context_line_but_no_line_and_col(self):
```

Comment:
```suggestion
    def test_has_context_line_but_not_both_line_and_col(self):
```

Slightly wordier, but describes all three cases in the test.